app/src/main/java/org/simple/clinic/sync/DataSync.kt

```diff
@@ -74,6 +74,12 @@ class DataSync @Inject constructor(
         .subscribe()
   }
 
+  fun fireAndForgetSync() {
+    syncTheWorld()
+        .subscribeOn(schedulersProvider.io())
+        .subscribe()
```

Comment:
Just a question, previously we had `onErrorComplete` for this call in the `LoginUserWithOtp` class. Why did we not use it over here? 🤔

Reply:
The individual syncs (See `sync(SyncGroup)`) themselves handle the errors internally, so this `onErrorComplete` was a redundant call.